R/community.R

```diff
@@ -696,15 +696,15 @@ as.hclust.communities <- function(x, hang = -1, use.modularity = FALSE,
 #' @rdname communities
 #' @family community
 #' @export
-as_phylo <- function(x, ...) {
-  UseMethod("as_phylo")
+as.phylo <- function(x, ...) {
+  UseMethod("as.phylo")
 }
 
 #' @rdname communities
-#' @method as_phylo communities
+#' @method as.phylo communities
```

Comment:
Do we need to use `s3_register()` in `.onLoad()` here?

Reply:
I'm a bit confused (and not very experienced with S3). That's what you suggested and what I added in `R/aaa-utils.R`, was that wrong?

Reply:
We don't need `#' @method`, I believe.